jhrbva/Caravan
# Ca 12/trip invitations view

To test:
Go to /invitation and you should see the details for a trip.
Make sure the server is running!
